mlir/include/mlir/Dialect/XeGPU/Transforms/Passes.td

```diff
@@ -38,4 +38,16 @@ def XeGPUSubgroupDistribute : Pass<"xegpu-subgroup-distribute"> {
   ];
 }
 
+def XeGPUBlocking: Pass<"xegpu-blocking"> {
+  let summary = "Instructionlize XeGPU ops";
```

Comment:
let's change that complex word "instructionlize" :)

Reply:
Fixed